drivers/wifi/nrf700x/zephyr/src/zephyr_disp_scan.c

```diff
@@ -47,6 +47,10 @@ int wifi_nrf_disp_scan_zep(const struct device *dev,
 
 	scan_info.scan_mode = AUTO_SCAN;
 	scan_info.scan_reason = SCAN_DISPLAY;
+	/* Wildcard SSID to trigger active scan */
+	scan_info.scan_params.num_scan_ssids = 1;
+	scan_info.scan_params.scan_ssids[0].img_ssid_len = 0;
+	scan_info.scan_params.scan_ssids[0].img_ssid[0] = 0;
 
```

Comment:
Remove img_ prefixes

Reply:
These are not introduced in this PR, and also present in common header files with RPU, so, left it untouched, once we clean those header files then we can do these changes.